### PR TITLE
1883881- Updated Powershell dedependencies

### PR DESCRIPTION
**Pull request by v-saikumart**
**Task name**: 
AzurePowerShellV4
AzurePowerShellV4
AzurePowerShellV5
AzurePowerShellV5

**Description**: Update dependencies on "utility-common-v2" package to fix high vulnerability issues
https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/1883881

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) Y

**Attached related issue:** (Y/N) N

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected
https://github.com/microsoft/azure-pipelines-tasks/files/7364975/PSTask.txt

### CODE REVIEW

File: Tasks/AzurePowerShellV4/azurepowershell.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import path = require('path');
 import os = require('os');
 import tl = require('azure-pipelines-task-lib/task');
 import tr = require('azure-pipelines-task-lib/toolrunner');
-import * as telemetry from 'azure-pipelines-tasks-utility-common/telemetry';
+//import * as telemetry from 'azure-pipelines-tasks-utility-common/telemetry';
```
Comment (AmrutaKawade) [inline Tasks/AzurePowerShellV4/azurepowershell.ts:6]: Remove this. DOn't just comment if its not getting used

File: Tasks/AzurePowerShellV4/package.json [hunk 2]
```diff
@@ -8,8 +8,8 @@
         "@types/node": "^10.17.0",
         "@types/q": "1.0.7",
         "azure-pipelines-task-lib": "^3.0.6-preview.0",
-        "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.0-preview.0",
-        "azure-pipelines-tasks-utility-common": "^3.0.0-preview.0"
+        "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.4",
+        "azure-pipelines-tasks-utility-common": "^3.0.3"
```
Comment (AmrutaKawade) [inline Tasks/AzurePowerShellV4/package.json:12]: You can remove this package from dependency as well.

File: Tasks/AzurePowerShellV5/azurepowershell.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import path = require('path');
 import os = require('os');
 import tl = require('azure-pipelines-task-lib/task');
 import tr = require('azure-pipelines-task-lib/toolrunner');
-import * as telemetry from 'azure-pipelines-tasks-utility-common/telemetry';
+//import * as telemetry from 'azure-pipelines-tasks-utility-common/telemetry';
```
Comment (AmrutaKawade) [inline Tasks/AzurePowerShellV5/azurepowershell.ts:6]: Remove

File: Tasks/AzurePowerShellV5/package.json [hunk 4]
```diff
@@ -8,8 +8,8 @@
         "@types/node": "^10.17.0",
         "@types/q": "1.0.7",
         "azure-pipelines-task-lib": "^3.0.6-preview.0",
-        "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.0-preview.0",
-        "azure-pipelines-tasks-utility-common": "^3.0.0-preview.0"
+        "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.4",
+        "azure-pipelines-tasks-utility-common": "^3.0.3"
```
Comment (AmrutaKawade) [inline Tasks/AzurePowerShellV5/package.json:12]: Remove

**Review by AmrutaKawade — CHANGES_REQUESTED** [review]

File: Tasks/AzurePowerShellV4/azurepowershell.ts (hunk 1 above)
Comment (v-saikumart) [reply Tasks/AzurePowerShellV4/azurepowershell.ts]: As suggested removed the commented code.

File: Tasks/AzurePowerShellV4/package.json (hunk 2 above)
Comment (v-saikumart) [reply Tasks/AzurePowerShellV4/package.json]: As suggested made the changes.

File: Tasks/AzurePowerShellV5/azurepowershell.ts (hunk 3 above)
Comment (v-saikumart) [reply Tasks/AzurePowerShellV5/azurepowershell.ts]: As suggested made the changes.

**Pull request merged**